packages/protocol/test/identity/federatedattestations.ts

```diff
@@ -886,42 +812,21 @@ contract('FederatedAttestations', (accounts: string[]) => {
     it('should succeed when >1 attestations are registered for (identifier, issuer)', async () => {
       const account2 = accounts[3]
       await signAndRegisterAttestation(identifier1, issuer1, account2, nowUnixTime, signer1)
-      const deleteAttestation = await federatedAttestations.deleteAttestation(
-        identifier1,
-        issuer1,
-        account2,
-        {
-          from: account2,
-        }
-      )
-      assertLogMatches2(deleteAttestation.logs[0], {
-        event: 'AttestationDeleted',
-        args: {
-          identifier: identifier1,
-          issuer: issuer1,
-          account: account2,
-        },
+      await checkStorageState(identifier1, issuer1, 1, account2, nowUnixTime, signer1, 0)
+      await federatedAttestations.deleteAttestation(identifier1, issuer1, account2, {
+        from: account2,
       })
+      await checkStorageState(identifier1, issuer1, 0, account1, nowUnixTime, signer1, 0)
```

Comment:
nice! nit: can you also add a `checkAttestationDeleted` to ensure the second one was deleted too?